applications/crossbar/priv/couchdb/schemas/access_lists.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_id": "access_lists",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": false,
+  "name": "ACL Entry",
+  "description": "Access Control List entries for device or account",
+  "additionalProperties": false,
+  "properties": {
+    "cidrs": {
+      "type": "array",
+      "required": true,
+      "name": "CIDR",
+      "description": "Classless Inter-Domain Routing IP notation for use on the access lists"
+    },
+    "order":{
+      "type":"string",
+      "enum":[
+        "AD",
```

Comment:
I don't think it is worth using `AD` and `DA`; it obfuscates the meaning. I support "allow,deny" and "deny,allow" as being acceptable and expected (esp since other software like Apache uses it).